frontend/app/src/pages/Connected/Home/index.js

```diff
@@ -36,46 +36,12 @@ const ConnectedHome = (props) => {
 
   return (
     <Page
-      //   metaDescKey={KEYS.whoAreWeBody}
+      metaDescKey={KEYS.whoAreWeConnectedBody}
       className="Connected ConnectedContent ConnectedHome"
       pageTitle="Junction 2020 Connected"
-      //   metaDescKey={KEYS.whoAreWeBody}
-      //   ogImageKey={MEDIA_KEYS.homePageHeaderImage}
+      ogImageKey={MEDIA_KEYS.connectedHeaderImage}
     >
       <div className="Connected-parallax">
-        <Helmet>
-          <meta charSet="utf-8" />
-          <title>Junction 2020 Connected</title>
-          <meta
-            name="keywords"
-            content="junction 2020 connected, connected, junction, junction 2020"
-          />
-          <meta name="title" content="Junction 2020 Connected" />
-          <meta property="og:title" content="Junction 2020 Connected" />
-          <meta name="twitter:title" content="Junction 2020 Connected" />
-          <meta
-            name="description"
-            content="Junction 2020 Connected is a new take on the established concept of a hackathon; participants all over the world can join physical locations hosted by Junction and other organizations, or participate in the event fully online."
-          />
-          <meta
-            property="og:description"
-            content="Junction 2020 Connected is a new take on the established concept of a hackathon; participants all over the world can join physical locations hosted by Junction and other organizations, or participate in the event fully online."
-          />
-          <meta
-            name="twitter:description"
-            content="Junction 2020 Connected is a new take on the established concept of a hackathon; participants all over the world can join physical locations hosted by Junction and other organizations, or participate in the event fully online."
-          />
-
-          <meta name="og:type" content="website" />
-          <meta
-            property="og:image"
-            content={require('../../../assets/images/photo-hub-visualisation.svg')}
-          />
-          <meta
-            name="twitter:image"
-            content={require('../../../assets/images/photo-hub-visualisation.svg')}
-          />
-        </Helmet>
```

Comment:
This deletes entire Helmet section? 🤔

Reply:
Yeah, because the Helmet section was already defined in PageHOC, and therefore it's possible to set the picture and description through Strapi Keys 😄  So this was basically really unnecessary to do in the first page.